node/tests/TestUtilities.ts

```diff
@@ -1756,3 +1757,42 @@ export async function transactionTest(
     responseData.push(["wait(1, 200)", 1]);
     return responseData;
 }
+
+/**
+ * This function gets server version using info command in glide client.
+ *
+ * @param addresses - Addresses containing host and port for the valkey server.
+ * @returns Server version for valkey server
+ */
+export async function getServerVersion(
+    addresses: [string, number][],
+    clusterMode = false,
+): Promise<string> {
+    let info = "";
+
+    if (clusterMode) {
+        const glideClusterClient = await GlideClusterClient.createClient(
+            getClientConfigurationOption(addresses, ProtocolVersion.RESP2),
+        );
+        info = getFirstResult(
+            await glideClusterClient.info({ sections: [InfoOptions.Server] }),
+        ).toString();
+    } else {
+        const glideClient = await GlideClient.createClient(
+            getClientConfigurationOption(addresses, ProtocolVersion.RESP2),
+        );
+        info = await glideClient.info([InfoOptions.Server]);
+    }
```

Comment:
close clients after use! Unclosed clients hang the test

Reply:
Updated.